apps/full-site-editing/full-site-editing-plugin/editor-site-launch/src/launch-steps/domain-step/index.tsx

```diff
@@ -12,26 +12,15 @@ import LaunchStep, { Props as LaunchStepProps } from '../../launch-step';
 import DomainPickerFSE from '../../../../editor-domain-picker/src/domain-picker-fse';
 import './styles.scss';
 
-const DomainStep: React.FunctionComponent< LaunchStepProps > = ( { onPrevStep, onNextStep } ) => {
-	const handleBack = () => {
-		onPrevStep?.();
-	};
-
+const DomainStep: React.FunctionComponent< LaunchStepProps > = ( { onNextStep } ) => {
 	return (
 		<LaunchStep className="nux-launch-domain-step">
 			<div className="nux-launch-step__header">
-				<div className="nux-launch-step__heading">
-					<h1 className="nux-launch-step__title">
-						{ __( 'Choose a domain', 'full-site-editing' ) }
-					</h1>
-					<p className="nux-launch-step__subtitle">
+				<div>
+					<Title>{ __( 'Choose a domain', 'full-site-editing' ) }</Title>
+					<SubTitle>
 						{ __( 'Free for the first year with any paid plan', 'full-site-editing' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Free for the first year with any paid plan' )` *ES Score: 11*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/better-message-for-conflicting-subdomain-zone)